generated/nodejs/google-cloud-node/packages/monitoring/README.md

```diff
@@ -0,0 +1,45 @@
+Google Monitoring API for Node.js
```

Comment:
This needs to be hand-edited to 'Stackdriver Monitoring API', please edit all the relevant places.